main.go

```diff
@@ -8,6 +8,11 @@ import (
 	log "github.com/Sirupsen/logrus"
 	"github.com/rancherio/go-machine-service/events"
 	"github.com/rancherio/go-machine-service/handlers"
+	_ "github.com/rancherio/go-machine-service/handlers/providers/amazonec2"
```

Comment:
@wlan0 I don't want to spend any more time on this. You aren't going to change it?